libqtile/backend/wayland/inputs.py

```diff
@@ -278,13 +281,23 @@ def _on_key(self, _listener: Listener, event: KeyboardKeyEvent) -> None:
 
         self.seat.keyboard_notify_key(event)
 
+    def set_num_caps_lock(self, numlock: bool, capslock: bool) -> None:
+        """Set the initial num/caps lock state of the keyboard."""
+        mask = ModifiersMask(self.keyboard)
+        if numlock:
+            mask.add("Mod2")
+        if capslock:
+            mask.add("Lock")
+        self.keyboard.notify_modifiers(mask)
```

Comment:
Could we get the current state of the modifiers and use that as the "default" mask, then do something like this?:
```py
if numluck is not None:
    if numluck:
        mask.add("Mod2")
    else:
        mask.remove("Mod2")
# ... same for capslock
```